driving.js

```diff
@@ -0,0 +1,10 @@
+let firstName = 'Jennifer ';
+let lastName = 'Lee ';
+let age = 15;
+let i = 16;
```

Comment:
since this isn't a counter, a better semantic name would be good (like legalDrivingAge or something)